packages/nbdime/package.json

```diff
@@ -50,8 +56,5 @@
     "rimraf": "^5.0.0",
     "ts-jest": "^29.1.0",
     "typescript": "^4.9.0"
-  },
-  "peerDependencies": {
-    "codemirror": "^5.0.0"
```

Comment:
do wee have peer dependencies on other codemirror packages?

Reply:
It seems that we don't need peer dependencies.